### PR TITLE
At home page implement history section, banner slider #6280

### PR DESCRIPTION
**Pull request by gautampanday**
At home page implement 
- changes relative images and content changed

### CODE REVIEW

File: theme/templates/index.html [hunk 1]
```diff
@@ -62,131 +76,117 @@ <h3 class="text-uppercase">What is Tryton?</h3>
...
+    </div>
+  </section>
+  <div class="mar_top4 clearfix"></div>
+  <section class="history">
+    <div class="container">
+      <div class="mar_top6"></div>
+      <div class="col-md-6 col-sm-6 col-xs-12">
+        <div class="col-md-3 col-sm-6 col-xs-12 text-center">
+         <img src="{{ STATIC }}images/history-languages.png" alt="" class="img img-responsive content-center-xs">
+         <p class="white-space">Languages Supported</p>
```
Comment (tarunbhardwaj) [inline theme/templates/index.html]: do you really need this new class for 2 words

**Comment by tarunbhardwaj** [comment]
Can you please move all end-users logos to a folder in images.

File: theme/templates/index.html (hunk 1 above)
Comment (gautampanday) [inline theme/templates/index.html]: yes for arranging at small size this will be required

File: theme/static/css/responsive-layouts.css [hunk 2]
```diff
@@ -34,11 +34,22 @@
...
 }
 
 @media only screen and (min-width: 768px) and (max-width: 992px) {
   .text-center-sm{
     text-align: center;
   }
+  .white-space{
+    white-space: nowrap;
+  }
+  .border-left-right{
```
Comment (tarunbhardwaj) [inline theme/static/css/responsive-layouts.css:50]: no-border-left-right-sm

File: theme/static/css/responsive-layouts.css [hunk 3]
```diff
@@ -34,11 +34,22 @@
...
+    border-left: 0px;
+    border-right: 0px;
+  }
 }
 
 @media only screen and (min-width: 768px) and (max-width: 992px) {
   .text-center-sm{
     text-align: center;
   }
+  .white-space{
```
Comment (tarunbhardwaj) [inline theme/static/css/responsive-layouts.css]: not sure why need this

File: theme/static/css/style.css [hunk 4]
```diff
@@ -16,6 +17,33 @@ a:hover {
   min-width: 113px;
 }
 
+.tryton-conf-banner {
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css:20]: create a common class of banners then add just classes for changing background

File: theme/templates/index.html [hunk 5]
```diff
@@ -14,13 +14,27 @@
...
+          <a class="btn btn-lg btn-primary" href="http://lanyrd.com/2014/tryton-unconference-leipzig/" target="_blank" role="button">{% trans %}See Complete Schedule{% endtrans %}</a>
         </div>
       </div>
       <div class="item">
-        <img src="{{ STATIC }}images/banner/slide2.jpg" class="sliderImg" alt="slide2" />
+        <div class="business-banner banner"></div>
         <div class="carousel-caption">
+          <h1>{% trans %}SECURE. STABLE. SCALABLE.{% endtrans %}</h1>
+          <p>{% trans %}Give your business a sturdy backbone with Tryton.{% endtrans %}</p>
+          <a class="btn btn-lg btn-primary" href="contact.html" role="button">Request a Demo</a>
```
Comment (tarunbhardwaj) [inline theme/templates/index.html]: trans

File: theme/static/css/style.css [hunk 6]
```diff
@@ -137,6 +163,10 @@ a:hover {
   margin: 0;
 }
 
+.pad-all20{
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css]: already there

File: theme/static/css/style.css [hunk 7]
```diff
@@ -382,4 +399,38 @@ a:hover {
...
   border-bottom: 2px solid #2283bc;
-}
\ No newline at end of file
+}
+
+.company-slider {
+  min-height: 240px;
+}
+
+.transition:hover img {
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css]: rename this class

**Pull request merged**